hugegraph-core/src/main/java/org/apache/hugegraph/backend/query/ConditionQueryFlatten.java

```diff
@@ -51,6 +51,22 @@ public static List<ConditionQuery> flatten(ConditionQuery query) {
     public static List<ConditionQuery> flatten(ConditionQuery query,
                                                boolean supportIn) {
         if (query.isFlattened() && !query.mayHasDupKeys(SPECIAL_KEYS)) {
+            Relations relations = new Relations();
+            List<Condition> noRelations = new ArrayList<>();
```

Comment:
rename to conditions?

Reply:
there are two type, the one is relation, and other, so maybe relation  name is more suitable.